src/os/webgl/abstractwebglrenderer.js

```diff
@@ -18,6 +18,18 @@ goog.require('os.webgl.IWebGLRenderer');
 os.webgl.AbstractWebGLRenderer = function() {
   os.webgl.AbstractWebGLRenderer.base(this, 'constructor');
 
+  /**
+   * The id string of the renderer; probably the same as the plugin
+   * @type {string|undefined}
+   */
+  this.id;
```

Comment:
This and `description` should probably be defined on `os.webgl.IWebGLRenderer`.

Reply:
I was concerned about a breaking change on the `IWebGLRenderer`. If I don't need to worry about that I can add all three properties and get rid of the random fallback for ids.

Reply:
If you're concerned about that (which I don't think I am), then you can just put it in another interface and have the class implement both interfaces.